rcamp/accounts/templates/account-request-create.html

```diff
@@ -93,7 +93,7 @@ <h4 class="modal-title">Definitions &amp; Form Help</h4>
         {% endif %}
             <div class="col-sm-12">
               <p id="ucb-login-help"><i class="fa fa-info-circle" aria-hidden="true"></i> UCB users use their CU IdentiKey username and password.</p>
-              <p id="csu-login-help"><i class="fa fa-info-circle" aria-hidden="true"></i> CSU users use their CSU EID as username. Password is <code>${csu_password},${duo_otp}</code> or <code>${csu_password},push</code>.</p>
+              <p id="csu-login-help"><i class="fa fa-info-circle" aria-hidden="true"></i> CSU users use their CSU EID as username. Password is <code>csu_password,duo_key</code> or <code>csu_password,push</code>.</p>
```

Comment:
What about:
```html
CSU users use their CSU EID as username, and CSU password followed by <code>,6-digit-duo-passcode</code> or <code>,push</code>.
```

I feel as if that removes most of the ambiguity created by removing the BASH variable syntax.